winforms/formatting-utility/vb/Form1.vb

```diff
@@ -29,14 +29,14 @@ Public Class Form1
         StatusBar.Items.AddRange(New ToolStripItem() {label})
 
         ' Get localized strings for user interface.
-        Me.Text = rm.GetString("WindowCaption")
```

Comment:
Your last commit modified line 23, which was better in its original form. Instead of
` Private Sub Form1_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load`
it should be
` Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load`

Reply:
@rpetrusha , Isn't ` Private Sub Form1_Load() Handles MyBase.Load` even better because `sender` and `e` aren't used in thoses events ?

It's even mentioned in the [VB Coding Conventions](https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/program-structure/coding-conventions#language-guidelines) article under `Relaxed Delegates in Event Handlers`

Reply:
I think I fixed it, in latest commit. I believe it should be
```VB
Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
```